src/EditorFeatures/CSharpTest/Debugging/ProximityExpressionsGetterTests.cs

```diff
@@ -61,7 +61,7 @@ static void Main(string[] args)
 }");
         var terms = CSharpProximityExpressionsService.GetProximityExpressions(tree, 245, cancellationToken: default);
         Assert.NotNull(terms);
-        AssertEx.Equal(["yy", "xx"], terms);
+        AssertEx.Equal((string[])["yy", "xx"], terms);
```

Comment:
what's going on here?  not loving that these IDE tests just got a lot more verbose (which goes against moving to collection exprs in the first place) :)

Reply:
Unfortunately xUnit introduced overloads that make these ambiguous now. I'm open to suggestions on how to approach this but it's hard to see how to reconcile the platform issues.

Reply:
This is a good example of where `Equals(ROSpan<T> span, T[] array)` won't fix all our issues. The `terms` value in this case is `IList<T>`.

Reply:
@333fred

Reply:
Do you know what it was using before? Was this an `IEnumerable<T>` vs `IEnumerable<T>` case?

Reply:
So, this is an `AssertEx` helper, not an `Assert` helper. What methods are actually ambiguous here?

Reply:
Sure, but this is not a general xUnit API, this is our own extension. So we can just add whatever helpers we need to make it work.